introduction.md

```diff
@@ -3,4 +3,6 @@ Office : Vensi
 Id No. : 72
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?